### DISCUSSION TITLE
Update files

### DISCUSSION THREAD

**Discussion by mojobnichols** [directus/directus#9073] [Feature Requests]
I would like to be able to update files.   I see I can for images (jpeg), but not for other files.   Seems like if it is not a image instead of showing the image having a button that says Replace would work.

**Comment by rijkvanzanten** [comment]
This is in regards to the file detail page in the app correct?

**Reply by mojobnichols** [reply]
Yes

**Reply by mojobnichols** [reply]
Is there any update on this it seems high on the discussion list and relatively easy to fix (use the same functionality as images has for pdf xlsx etc)?   Should I click create issue from it?  As our site is getting closer to launch it is becoming something we will have to find work around but it seems like you should be able to update a file instead of deleting it and uploading it.   Not to mention the ramifications deleting and uploading ramification creates if file name changes. 

Thank you.

**Reply by rijkvanzanten** [reply]
Yeah! The fact that the replace operation disappears when the file is not an image feels like a bug to me if anything. I've moved it to an issue here: https://github.com/directus/directus/issues/11898

**Comment by br41nslug** [comment]
i believe this was implemented
